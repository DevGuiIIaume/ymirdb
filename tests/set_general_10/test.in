set a 1
set b 2
set b a 2 a
list entries
pop b
pop b
forward a
forward b
backward a
backward b
set b a a
forward a
forward b
backward a
backward b
bye
