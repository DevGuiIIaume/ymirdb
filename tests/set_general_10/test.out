> ok

> ok

> ok

> b [a 2 a]
a [1]

> a

> 2

> nil

> a

> b

> nil

> ok

> nil

> a

> b

> nil

> bye
